catalog/imaging/imaging-plugin-nitf/pom.xml

```diff
@@ -119,6 +119,16 @@
             <artifactId>commons-lang3</artifactId>
             <version>${commons-lang3.version}</version>
         </dependency>
+        <dependency>
+            <groupId>ddf.catalog.async</groupId>
+            <artifactId>catalog-async-data</artifactId>
```

Comment:
❓Instead of having a dependency on this module here, embed `catalog-async-data` below and new up the objects in the `NitfPostProcessPlugin` class, should we consider adding a factory interface (or interfaces) to `catalog-async-plugin-api` to create the `Impl` objects? This is something that @pklinef has brought up a couple of times and I think it would help reduce coupling between modules and bundles.

Reply:
I like this idea but don't think this is the appropriate PR to do it.